Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackFeatureProcessor.cpp

```diff
@@ -143,5 +143,24 @@ namespace AZ
                 AZ::TickBus::Handler::BusDisconnect();
             }
         }
+
+        void EditorModeFeatureProcessor::SetEnableRender(bool enableRender)
+        {
+            m_enableRender = enableRender;
+            
+            if (!m_editorStatePassSystem)
+            {
+                return;
+            }
+
+            const auto templateName = Name(m_editorStatePassSystem->GetParentPassTemplateName());
+
+            auto passFilter = AZ::RPI::PassFilter::CreateWithTemplateName(templateName, GetParentScene());
+            AZ::RPI::PassSystemInterface::Get()->ForEachPass(passFilter,  [this](RPI::Pass* pass) -> RPI::PassFilterExecutionFlow
```

Comment:
suggestion: Rather than capturing this we could do

```
[enableRender = m_enableRender]
```

in the capture clause and use that as that is all we need from the surrounding code